crates/re_data_store/src/entity_tree.rs

```diff
@@ -88,22 +88,47 @@ pub struct SubtreeInfo {
     ///
     /// ⚠ Auto-generated instance keys are _not_ accounted for. ⚠
     pub time_histogram: TimeHistogramPerTimeline,
+
+    /// Number of bytes used by all arrow data in this tree (ignores overhead from book-keeping, schemas, etc).
```

Comment:
The size of the cell _does_ take into account the copy of the arrow schema that is stored in the cell